packages/playground/src/layouts/RepoLayout.tsx

```diff
@@ -7,10 +7,10 @@ const RepoLayout: React.FC = () => {
 
   return (
     <div>
-      <header style={{ background: '#333', padding: '20px', height: '56px' }}>
+      <header className="p-5 h-16 bg-black">
         <h1>Repository {repoId}</h1>
       </header>
-      <header style={{ background: '#222', padding: '20px' }}>
+      <header className="p-5 h-16" style={{ lineHeight: 1}}>
```

Comment:
do we need to define the `lineHeight`? if so, let's not use inline styles.. use TW classes instead

Reply:
deleted line height in advance